managed/src/main/java/com/yugabyte/yw/models/Backup.java

```diff
@@ -224,4 +228,36 @@ public static boolean existsStorageConfig(UUID customerConfigUUID) {
         .collect(Collectors.toList());
     return backupList.size() != 0;
   }
+
+  public static Set<Universe> getUniverses(UUID customerConfigUUID) {
```

Comment:
I've just noticed - why such function name? It's completely unclear what it does.